src/bentoml/_internal/configuration/v1/default_configuration.yaml

```diff
@@ -66,23 +91,38 @@ runners:
       response_content_length: true
       response_content_type: true
   metrics:
-    enabled: True
+    enabled: true
     namespace: bentoml_runner
-  timeout: 300
-
 tracing:
-  type: zipkin
+  exporter_type: ~
   sample_rate: ~
   excluded_urls: ~
+  timeout: ~
+  max_tag_value_length: ~
   zipkin:
```

Comment:
Since we are refactoring here, shall we use list here for exporters?

Reply:
I can follow up with a PR. Cna you explain more about multi-exporters?